repo/packages/C/chronos/0/marathon.json

```diff
@@ -15,7 +15,7 @@
   "container": {
     "type": "DOCKER",
     "docker": {
-      "image": "mesosphere/chronos:chronos-2.3.2-0.1.20150207000917.ubuntu1404-mesos-0.22.0-1.0.ubuntu1404",
+      "image": "mesosphere/chronos:chronos-2.3.3-1.0.1.ubuntu1404-mesos-0.22.0-1.0.ubuntu1404”,
```

Comment:
I'm not sure why the quote character at the end of this line would have changed (editor maybe?), but the change should probably be reverted so that the quote characters are consistent across the document.